src/payment-flows/checkout.js

```diff
@@ -101,6 +101,16 @@ function getContext({ win, isClick } : {| win : ?(CrossDomainWindowType | ProxyW
     return CONTEXT.IFRAME;
 }
 
+function getDimensions(fundingSource : string) : Object {
```

Comment:
Please avoid using the generic `Object` type.
```suggestion
function getDimensions(fundingSource : string) : {| width : number, height : number |} {
```

Reply:
fixed it